.github/workflows/c_linter.yml

```diff
@@ -0,0 +1,16 @@
+name: LintCAPIUsage
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
```

Comment:
Most of the other workflows have some code here to make sure the workflow doesn't run on forks unless someone explicitly opts into that by commenting out a line.